js_course/test-2.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('test-2', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').click();
+  await page.locator('[data-test="username"]').fill('standard_user');
+  await page.locator('[data-test="password"]').click();
+  await page.locator('[data-test="password"]').fill('secret_sauce');
+  await page.locator('[data-test="login-button"]').click();
+  await page.locator('[data-test="add-to-cart-sauce-labs-backpack"]').click();
```

Comment:
Probably, you need to add random products
From task - `Add several random products to the Shopping Cart`

Reply:
I don't see such task, I need to add the first one